codegen/smithy-ruby-codegen-test/model/component-test/paginators.smithy

```diff
@@ -44,6 +44,11 @@ structure ResultWrapper {
     __123nextToken: String
 }
 
-list Items {
-    member: String
+//list Items {
+//    member: String
+//}
```

Comment:
These are commented out - what's the reason to keep them? Just curious about the intent!

Reply:
Good catch. It was from testing different types.